File: LISTA-02(JAVA)/Q1.java
class Circulo {
    private double r = 0;
    public void setRaio (double v) {
        if (v >= 0) this.r = v;
        else throw new IllegalArgumentException("Valor negativo é inválido");
    }
    public double getRaio() {
        return this.r;
    }
    public double calcArea() {
        return 3.14 * this.r * this.r;
    }
    public double calcCircun() {
        return 2 * 3.14 * this.r;
    }
}

public class Q1 {
    public static void main(String[] args) {
        Circulo x = new Circulo();
        x.setRaio(-10);
        System.out.println("A área é " + x.calcArea() + " e a circunferência é : " + x.calcCircun());
    }
}

File: LISTA-02(PYTHON)/Q1.py
class Circulo :
    def __init__(self) :
        self.__r = 0.0
    def set_raio(self, v) :
        if v >= 0 :
            self.__r = v
        else :
            raise ValueError("Valor negativo é inválido")
    def get_raio(self) :
        return self.__r
    def calc_area(self) :
        area = 3.14 * self.__r * self.__r
        return area
    def calc_circun(self) :
        circunferencia = 2 * 3.14 * self.__r
        return circunferencia
    
class UI :
    @staticmethod
    def main() :
        x = Circulo()
        x.set_raio(-10)
        print(x.calc_area(), x.calc_circun())

UI.main()